docs/data/query.md

```diff
@@ -110,50 +110,41 @@ You do not need to perform any additional configuration when [querying data in t
    viam data database configure --org-id=<YOUR-ORGANIZATION-ID> --password=<NEW-DBUSER-PASSWORD>
    ```
 
-   This command configures a new database user for your org for use with data query.
+   This command configures a new database user for your org for use with data query, and sets the password.
+   If you have already run this command before, this command instead updates the password to the new value you set.
 
-1. Determine the hostname for your organization's MongoDB Atlas Data Federation instance by running the following command with the organization's `org-id` from step 2:
+1. Determine the connection URI for your organization's MongoDB Atlas Data Federation instance by running the following command with the organization's `org-id` from step 2:
 
    ```sh {class="line-numbers linkable-line-numbers"}
    viam data database hostname --org-id=<YOUR-ORGANIZATION-ID>
    ```
 
-   This command returns the `hostname` (including database name) to use to connect to your data store on the Viam organization's MongoDB Atlas instance.
+   This command returns the hostname (including database name) and connection URI to use to connect to your data store on the Viam organization's MongoDB Atlas instance.
```

Comment:
Doesn't URI include hostname?

Reply:
Engineering chose to retain the original output alongside the additional new output. Ostensibly, its so that users used to or expecting the previous output still have access to it, while adding additional data output to benefit those users who seek the new form of this data output.

It is also beneficial for situations where the user does not need the URI (legacy SQL clients that ask for server hostname, database name, username, and password in 4 separate GUI fields. Like [DBeaver](https://dbeaver.com/wp-content/uploads/2022/03/Screen-Shot-2022-03-03-at-11.52.55-AM.png), etc). Saying something like "take the connection URI from the command above and delete a bunch of text we don't need to get at the text we do need" would be more cumbersome for them then simplifying output to neat single data return for us.